src/app/modules/item/components/item-header/access-code-view/access-code-view.component.ts

```diff
@@ -13,7 +13,7 @@ import { ItemData } from '../../../services/item-datasource.service';
 export class AccessCodeViewComponent {
 
   @Input() itemData?: ItemData;
-  @Output() reload = new EventEmitter<void>();
+  @Output() reloadItemPage = new EventEmitter<void>();
```

Comment:
In yesterday's remark, I added "which may be used in different context". So not sure it should be tight to an item page (possibly with some modifications of the component). Here what we just want to know is that there were a change and so that the parent component can decide what to do. So we could use for instance: "codeAccepted" or "membershipChanged" or ...